### ISSUE TITLE
Some questions about the evalution metrics

### ISSUE THREAD

**Issue by henanjun** [anilbatra2185/road_connectivity#28]
Hi,
I notice that you used a buffer of 4 pixels in your evaluations in your paper. And I noticed that there is function called "relaxed_f1". However, I find you do not use the  "relaxed_f1"  function in your main training or test stage (i.e., train_mtl.py).

So my question is that the results reported in your paper used "the bufffer evalution" or not? It confused me.

**Comment by anilbatra2185** [comment]
Thanks @henanjun for raising this question. I used the buffer size of 4 as mentioned in the paper, however, I specifically used the "realxed_f1" function to compute the metrics on validation set after training is completed using the best saved model. There are two IoU (relaxed and accurate) metrics are reported and relaxed uses buffer of 4 pixels while accurate IOU is simple pixel based.

I hope it helps!

Regards
Anil

**Comment by henanjun** [comment]
I see, Thank you very much for your kind reply.

**Comment by henanjun** [comment]
I see, Thank you very much for your kind reply. @anilbatra2185

**Comment by anilbatra2185** [comment]
Closing this issue as you are able to find the solution. Thanks!
